internal/audit/auditlog.go

```diff
@@ -28,16 +28,24 @@ import (
 // results.
 func LogResults(formatter log.Formatter, packageCount int, coordinates []types.Coordinate, invalidCoordinates []types.Coordinate, exclusions []string) int {
 	vulnerableCount := 0
+	exclusionCount := 0
 
 	for _, c := range coordinates {
 		c.ExcludeVulnerabilities(exclusions)
 	}
 
 	var auditedCoordinates []types.Coordinate
 	var vulnerableCoordinates []types.Coordinate
+	var excludedVulnerabilities []types.Vulnerability
 
 	for i := 0; i < len(coordinates); i++ {
 		coordinate := coordinates[i]
+		for _, v := range coordinate.Vulnerabilities {
+			if v.Excluded {
+				exclusionCount++
```

Comment:
Minor: You can likely just do `len(excludedVulnerabilities)` to skip having another variable for this